Build/Build.cs

```diff
@@ -241,12 +263,15 @@ class Build : NukeBuild
         });
 
     Target Pack => _ => _
-        .DependsOn(ApiChecks)
-        .DependsOn(TestFrameworks)
-        .DependsOn(UnitTests)
-        .DependsOn(CodeCoverage)
-        .DependsOn(CalculateNugetVersion)
+        .DependsOn(
+            ApiChecks,
+            TestFrameworks,
+            UnitTests,
+            CodeCoverage,
+            CalculateNugetVersion
+        )
         .OnlyWhenDynamic(() => RunAllTargets || HasSourceChanges)
+        .Produces(ArtifactsDirectory / "*.nupkg")
```

Comment:
What does this do?

Reply:
Oh.. nothing anymore..
If we would use the yaml code generation, this would generate the "upload artifacts" step